SparseGrids/TasmanianSparseGrid.hpp

```diff
@@ -1200,6 +1207,13 @@ class TasmanianSparseGrid{
     void setSurplusRefinement(double tolerance, int output, std::vector<int> const &level_limits);
     //! \brief Overload that uses array for the level limits.
     void setSurplusRefinement(double tolerance, int output, const int *level_limits = nullptr);
+    /*!
+     * \brief Call setSurplusRefinement() for Sequence and Global grids with a sequence rule and then getNeededPoints().
+     */
+    std::vector<double> getSurplusRefinement(double tolerance, int output, std::vector<int> const &level_limits) {
```

Comment:
This is OK. Just add it to the Doxygen at the top of the file where different methods are split into categories.

Reply:
Done.